QQE-378.md

```diff
@@ -0,0 +1,43 @@
+# QQE-378 - HTTP module / REST use-cases - Development topic compression Brotli4j
+## Links
+Jira : [https://issues.redhat.com/browse/QQE-378](https://issues.redhat.com/browse/QQE-378)
+
+PR:  TODO
+## Scope of the testing
+ - Integration of the Brotli4J compression library with Quarkus, particularly in the context of REST endpoints using RestEasy Reactive.
+ - To use Brotli4J compression we will need a custom Server configuration
+   * Utilize `io.vertx.core.http.HttpServerOptions` to enable compression support.
+   * Add the Brotli compressor using `.addCompressor(io.netty.handler.codec.compression.StandardCompressionOptions.brotli())`.
+   * Ensure that the server is configured to respect the Accept-Encoding header to automatically compress the response body with one of the supported compressions and send it back to the client.
+  ### Test scenarios 
+
+- Compression Enabled, Response Compressed
+   *    Set up a Quarkus application with the Brotli4J compression library
+   *    Configure the HTTP server to enable compression using io.vertx.core.http.HttpServerOptions
+   *    Add the Brotli compressor using `.addCompressor(io.netty.handler.codec.compression.StandardCompressionOptions.brotli())`
```

Comment:
Sorry, this is bit implementation specific, I have no idea what it means. Can you use words instead of some methods that I know nothing about to describe what is your intention behind using this compressor? In the TP please (not in the comment)